datadog_checks_dev/datadog_checks/dev/tooling/manifest_validator/validator.py

```diff
@@ -345,6 +346,22 @@ def validate(self, check_name, decoded, fix):
             self.fail(output)
 
 
+class SupportedOSValidator(ManifestValidator):
+    """If an integration contains python or logs configuration, the supported_os field should not be empty."""
+
+    def validate(self, check_name, decoded, _):
+        supported_os = decoded.get('supported_os')
+        check_has_logs = has_logs(check_name)
+        check_has_python = is_package(check_name)
+
+        if check_name in LOGS_EXCLUDE_LIST:
```

Comment:
Good point, I wasn't sure if that was still necessary since `LogsCategoryValidator()` is already checking that.

Reply:
I think it would catch an error in the case where an integration in `LOGS_EXCLUDE_LIST` like `cassandra_nodetool` has its `supported_os`'s removed

Reply:
Oh okay, I think I understand what you mean now. Do you mean I should these remove lines:

```
if check_name in LOGS_EXCLUDE_LIST:
    return
```
so that it doesn't skip checking for `supported_os` of the excluded logs integrations that may have python?

Reply:
You're right, I'll remove those lines because some of the integrations in `LOGS_EXCLUDE_LIST` still have python.